core/new-gui/src/app/workspace/service/save-workflow/save-workflow.service.ts

```diff
@@ -0,0 +1,86 @@
+import { Injectable } from '@angular/core';
+import { WorkflowActionService } from '../workflow-graph/model/workflow-action.service';
+import { Observable } from '../../../../../node_modules/rxjs';
+import { OperatorLink, OperatorPredicate, Point } from '../../types/workflow-common.interface';
+import { OperatorMetadataService } from '../operator-metadata/operator-metadata.service';
+
+
+export interface SavedWorkflow {
+  operators: OperatorPredicate[];
+  operatorPositions: {[key: string]: Point | undefined};
+  links: OperatorLink[];
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SaveWorkflowService {
+
+  private static localStorageKey: string = 'workflow';
+
+  constructor(
+    private workflowActionService: WorkflowActionService,
+    private operatorMetadataService: OperatorMetadataService
+  ) {
+    this.handleAutoSaveWorkFlow();
+    this.operatorMetadataService.getOperatorMetadata().subscribe(metadata => {
+      if (metadata.operators.length !== 0) {
+        this.loadWorkflow();
+      }
+    });
+  }
+
+  public loadWorkflow(): void {
```

Comment:
Add comments about this method, including the description of what kinds of information is loaded from the local storage.